content/posts/chapelcon25-retro/index.md

```diff
@@ -0,0 +1,103 @@
+This is backed by the technical story of Julia's relentless focus on composability, combined with the non-technical focus on sustainable, open-source development.
+Lessons for the Chapel community and inspiration for libraries and modules abound.
+Check it out [here](https://youtu.be/5ornbvwZJp8).
+
+### More Contributors Than Ever
+
+This year's ChapelCon had a record-breaking 20 presentations from community members across two days.
+The talks covered a wide range of topics, from machine learning and AI, to novel graph algorithms, to performance comparisons, and language interoperability. 
+While there are too many for me to cover here (I encourage you to check them all out [on YouTube](https://www.youtube.com/playlist?list=PLuqM5RJ2KYFgTmbn3PmOocfu5Sh2wEcAb)), I'd like to bring attention to some that were especially enjoyed by the community.
+
+One contribution that succinctly demonstrated Chapel's strengths in delivering both performance and productivity was [Mohammad Dindoost's talk on HiPerMotif](https://youtu.be/acvyLakS6gA).
+HiPerMotif is a hybrid parallel algorithm for identifying subgraph "motifs" within large-scale property graphs. 
+Implemented using Arache (an Arkouda/Chapel-based graph library), it provides up to 66x speedup compared to state-of-the-art methods, and processes graphs large enough to cause memory failures in other technologies. 
+
+The second highlight came from Daniel Fedorin and demonstrates the expansive power of Chapel's type system. 
+Using Chapel's compile-time `param` values and a bit of clever thinking, his approach makes it possible to encode complex data structures and specialized functions, all at compile time. 
+This has wide-reaching implications, including eliminating runtime overhead and expanding compile-time error checking for functions like `printf`. 
+Check out the recording [here](https://youtu.be/-x1zYQSuaT4).
+
+Third, from our very own PC chair Luca Ferranti, was a talk on [automatic differentiation in Chapel](https://youtu.be/ioqxdmSprBM).
+Useful in domains including machine learning, scientific computing, and optimization, automatic differentiation computes derivatives without the numerical error of finite-difference or the complexity of symbolic differentiation. 
+The `ForwardModeAD` library uses Chapel's operator overloading to support derivatives, gradients, jacobians, and more, all in a composable way.
+The cherry on top? A Chapel intergration for Enzyme, a library for automatic differentation at the LLVM-level!
+
+Finally, closing out the conference, was [Iain Moncrief's talk on his machine learning library ChAI](https://youtu.be/bHCrfHNkW6E).
+ChAI offers a set of tools to support high- or low-level ML programming, defining/loading existing models, and distributed inference.
+Integrated with PyTorch, ChAI supports developers at all levels, from those looking to load existing models as black boxes, build their own models entirely from scratch, or some combination of the two.
```

Comment:
This "from" feels dangling in that there's not a "to" following it.  Adding "to" is complicated by the fact that thelist includes three things where the third is "both the first", so maybe "whether for users looking to.."?  Or there are other ways to fix…